docs/content.rst

```diff
@@ -71,22 +71,31 @@ Metadata syntax for Markdown posts should follow this pattern::
     This is the content of my super blog post.
 
 You can also have your own metadata keys (so long as they don't conflict with
-reserved metadata keywords) for use in your python templates. The following is
+reserved metadata keywords) for use in your templates. Following is
```

Comment:
I think `Following is` -> `This is` would parse easier

Reply:
Didn't quite agree that was clearer, so I found another phrasing that might be easier to parse in https://github.com/getpelican/pelican/pull/2620/commits/f0617a53dc253432077fe3acbcd8d8bd87f636ab.